Resources/config/services.xml

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0" ?>
+
+<container xmlns="http://symfony.com/schema/dic/services"
+           xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+           xsi:schemaLocation="http://symfony.com/schema/dic/services
+        http://symfony.com/schema/dic/services/services-1.0.xsd"
+>
+    <services>
+        <service id="doctrine_bundle.migrations_collector"
+                 class="Doctrine\Bundle\MigrationsBundle\Collector\MigrationsCollector"
+                 public="true"
+        >
+            <argument type="service" id="doctrine.dbal.default_connection"/>
```

Comment:
There is absolutely no guarantee that a connection named ``default`` exists.

And please inject the ManagerRegistry to keep the connection service lazy. This way, you will be able to avoid forcing the initialization of the connection just because the profiler